awsm/framework/framework.py

```diff
@@ -550,21 +545,26 @@ def run_awsm_daily_ops(config_file):
         if idd > 0:
             # find previous output file
             prev_day = sd - pd.to_timedelta(1, unit='D')
-            prev_out = os.path.join(prev_out_base,
-                                    'run{}'.format(prev_day.strftime(fmt_day)),
-                                    'snow.nc')
+            prev_snow = os.path.join(
+                base_path,
+                'run{}'.format(prev_day.strftime(fmt_day)),
+                'snow.nc'
+            )
             # reset if running the model
             if new_config.cfg['awsm master']['model_type'] is not None:
                 new_config.raw_cfg['files']['init_type'] = 'netcdf_out'
-                new_config.raw_cfg['files']['init_file'] = prev_out
+                new_config.raw_cfg['files']['init_file'] = prev_snow
 
             # if we have a previous storm day file, use it
-            prev_storm = os.path.join(prev_data_base,
-                                      'data{}'.format(
```

Comment:
This `data{}` path was from a previous AWSM version and missed during the update.